browser/src/Services/Terminal.ts

```diff
@@ -15,26 +13,21 @@ export const activate = (
     configuration: Configuration,
     editorManager: EditorManager,
 ) => {
-    const getTerminalCommand = () => {
```

Comment:
Is this function only used in this scope?

Reply:
Yes, it's not exported or anything